Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
-import './App.css';
+import './css/index.css';
+import NumberButton from './components/ButtonComponents/NumberButton.js';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay.js';
+import ActionButton from './components/ButtonComponents/ActionButton.js';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+      <div className="appContainer">
+      <CalculatorDisplay text="0" />
+      <ActionButton text="clear" style="symbolStyle" />
```

Comment:
the code could be made drier by judicious use of map and a number/operator array.